src/test/rpc_tests.cpp

```diff
@@ -62,7 +62,7 @@ BOOST_FIXTURE_TEST_CASE(rpc_addmultisig, TestNetFixture)
     string short1(address1Hex, address1Hex+sizeof(address1Hex)-2); // last byte missing
```

Comment:
Yeah unless I'm mistaken that 1 comment is also wrong but otherwise this PR seems safe enough.